controller/Lists.php

```diff
@@ -595,6 +555,28 @@ private function createRemoteSourceContext(RdfClass $collectionClass): RemoteSou
         return new RemoteSourceContext($parameters);
     }
 
+    private function createListElementsFinderContext(
+        core_kernel_classes_Class $listClass,
+        bool $isRemote
+    ): ListElementsFinderContext {
+        $parameters = [
+            ListElementsFinderContext::PARAMETER_TYPE => $isRemote
+                ? ListElementsFinderProxy::TYPE_REMOTE
```

Comment:
Could not we use the RDF constants instead?

Reply:
Which one?

Reply:
The ones you use inside specifications

Reply:
I can check for list (remote or not) inside this `Finder`.
Wdyt?